### PR TITLE
Corrected flow for NetCat replcaemnet module

### PR DESCRIPTION
**Pull request by darkdrgn2k**
replacement of netcat converted to dialog question, and moved to more order after "extra tools" also converted to a module

"text installer" better descibed as "dialog ascii"

### CODE REVIEW

File: scripts/extra-tools-netcat/install [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (makeworld-the-better-one) [inline scripts/extra-tools-netcat/install]:
```suggestion
#!/usr/bin/env bash
```

File: scripts/extra-tools-netcat/install [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Replace netcat with IPv6 supported one
+# see https://unix.stackexchange.com/questions/457670/netcat-how-to-listen-on-a-tcp-port-using-ipv6-address
```
Comment (makeworld-the-better-one) [inline scripts/extra-tools-netcat/install]:
```suggestion
# See https://unix.stackexchange.com/questions/457670/netcat-how-to-listen-on-a-tcp-port-using-ipv6-address
```

File: scripts/extra-tools-netcat/install [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```
Comment (makeworld-the-better-one) [inline scripts/extra-tools-netcat/install]:
```suggestion
set -e
```
Should this be here?

File: scripts/install2 [hunk 4]
```diff
@@ -291,6 +278,7 @@ if [ "$WITH_PROMETHEUS_SERVER" == true ]; then
     askModule "WITH_GRAFANA" "Grafana"
 fi
 askModule "WITH_EXTRA_TOOLS" "Non-Essential Tools Useful for Network Analysis"
+askModule "WITH_EXTRA_NETCAT" "Replacement netcat that supports IPv6"
```
Comment (makeworld-the-better-one) [inline scripts/install2]:
```suggestion
askModule "WITH_EXTRA_NETCAT" "Change netcat to a version that supports IPv6"
```

Comment (darkdrgn2k) [reply scripts/install2]: Prompt prepends "INSTALL" in front of text.

`Install change netcat` does not read well

**Pull request merged**